### PR TITLE
MLOPS-336 Make MS2DeepScore and Spec2Vec outputs consistent

### PR DESCRIPTION
**Pull request by pedrocwb**
- Make sure to lowercase the include_metadata
- Sort columns of the endpoint.py output

### CODE REVIEW

File: omigami/endpoint.py [hunk 1]
```diff
@@ -9,13 +9,14 @@
...
     "compound_name",
+    "inchikey_inchi",
+    "inchikey_smiles",
     "instrument",
     "parent_mass",
-    "inchikey_smiles",
-    "inchikey_inchi",
+    "smiles",
 }
+
```
Comment (pedrocwb) [inline omigami/endpoint.py:19]: just sorted the keys

File: versioneer.py [hunk 2]
```diff
@@ -301,11 +300,13 @@ def get_root():
...
-               "its immediate directory (like 'python setup.py COMMAND'), "
-               "or in a way that lets it use sys.argv[0] to find the root "
-               "(like 'python path/to/setup.py COMMAND').")
+        err = (
+            "Versioneer was unable to run the project root directory. "
+            "Versioneer requires setup.py to be executed from "
+            "its immediate directory (like 'python setup.py COMMAND'), "
+            "or in a way that lets it use sys.argv[0] to find the root "
+            "(like 'python path/to/setup.py COMMAND')."
+        )
```
Comment (pedrocwb) [inline versioneer.py:309]: black format

**Review by dianacarvalho1 — APPROVED** [review]
ah then we decide the order of the metadata, not the user.. fine by me!  👍🏼

**Comment by pedrocwb** [comment]
> ah then we decide the order of the metadata, not the user.. fine by me! 👍🏼

Yeah. Let's not trust the user :P

**Review by tamaralipows — APPROVED** [review]

**Pull request merged**